apps/medusa-be/CLAUDE.md

```
@@ -3,710 +3,412 @@
+pnpm test:integration:modules               # module tests (isolated)
+pnpm test:integration:http                  # HTTP/e2e tests
 npx medusa exec ./src/scripts/my-script.ts  # run script
-npx medusa db:generate MODULE             # gen migration
+npx medusa db:generate MODULE               # gen migration
 ```
 
+**Feature-flagged migrations:** Enable `FEATURE_*` env var before `make medusa-generate-migration MODULE=my_module`.
+
 ## Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)